modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,252 @@
+                    name: {{ .Service.PortName }}
+----
+
+[calloutlist]
+
+.. Hosts or domains used for mapping the public endpoints configured on the Mule app deployment.
+.. One or more Kubernetes routing resources, such as an Ingress resource.
+.. The Runtime Fabric-generated unique routing resource name. Use this placeholder to make sure the Kubernetes resource name meets the uniqueness constraint. If you don’t use this placeholder, make sure the resource name is unique.
+.. Host or domain from the Mule app's public URL.
+.. Path from the Mule app's public URL. 
+
+
+=== Example for OpenShift Routes
+This example shows an HTTP route-template resource for OpenShift routes:
+
+----
+apiVersion: rtf.mulesoft.com/v1
+kind: HTTPRouteTemplate
+metadata:
+  name: test-template-routes
+  namespace: rtf
+spec:
+  baseEndpoints:
+    - https://demo.example.com
+  resources:
+    - |
+apiVersion: v1
+kind: Route
+metadata:
+  name: {{ .ResourceName }}
+  namespace: {{ .Namespace }}
+spec:
+  host: {{ .Host }}
+  path: {{ .Path }}
+  to:
+    kind: Service
+    name: {{ .Service.Name }}
+  tls:
+    termination: edge
+    key: |-
+      -----BEGIN PRIVATE KEY-----
+      [...]
+      -----END PRIVATE KEY-----
+    certificate: |-              
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE-----
+    caCertificate: |-
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE-----
+
+----
+
+=== Example for Kubernetes Gateway API
+
+This example shows an HTTP route-template resource for Kubernetes Gateway API (HTTPRoute):
+
+----
+apiVersion: rtf.mulesoft.com/v1
+kind: HttpRouteTemplate
+metadata:
+  name: k8s-gateway-api-cr
+  namespace: rtf
+spec:
+  baseEndpoints:
+    - https://demo.example.com
+  resources: |-
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: {{ .ResourceName }}
+  namespace: {{ .Namespace }}
+spec:
+  parentRefs:
+  - kind: Gateway
+    name: cluster-gateway
+  hostnames:
+  - {{ .Host }}
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: {{ .Path }}
+    backendRefs:
+    - name: {{ .Service.Name }}
+      port: {{ .Service.Port }}
+
+----
+
+[NOTE]
+You are responsible for the configuration of the Kuberneates Gateway resources and allowing HTTP Routes from the route's namespace to be attached.
+
+
+== Using TLS in the HTTP Route-Template Custom Resource
+
+To use TLS in the routing resources in an HTTP route template, create a TLS secret in the Runtime Fabric namespace. For details, refer to the https://kubernetes.io/docs/concepts/services-networking/ingress/#tls[Kubernetes TLS] documentation.
+
+To configure the Runtime Fabric to manage the secrets lifecycle (Create, Update, and Delete) across all application namespace, use the `rtf.mulesoft.com/synchronized: "true"` label on the TLS secret. For example:
+
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: testsecret-tls
+  namespace: <agent-namespace>
```

Comment:
`namespace: rtf`. We haven't talked about `<agent-namespace>` anywhere else on this page